templates/worker/00-worker/_base/files/system-conf.yaml

```diff
@@ -0,0 +1,9 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/systemd/system.conf.d/origin-accounting.conf"
```

Comment:
`kubelet-cgroup-accounting.conf`

Can you add a description into the system unit so people know why this is there?